CHANGELOG.md

```diff
@@ -3,6 +3,12 @@
 *The format of this document is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).*
 
+## [v1.1.4](https://github.com/RaspberryPiFoundation/Bits/compare/v1.1.4...v1.1.5) - 2018-04-08
```

Comment:
Does this need bumping to `v1.1.5`

Reply:
and the date to 04-09 :)

Reply:
Yep. One sec! Bit rushed, should have been more careful! I suppose this is what reviews are for :)